### DISCUSSION TITLE
Guide to linking big data

### DISCUSSION THREAD

**Discussion by hunsicke** [moj-analytical-services/splink#1684] [Q&A] [answered]
Hi, I am looking for the "Guide to linking big data" that is mentioned here: https://moj-analytical-services.github.io/splink/topic_guides/performance/optimising_spark.html.

The link seems to be broken.  Is this still available?

Thanks!

**Comment by ADBond** [comment, marked as answer]
I believe the link is supposed to be to [this page](https://moj-analytical-services.github.io/splink/topic_guides/performance/drivers_of_performance.html).
